setup.py

```diff
@@ -57,7 +57,7 @@ def run(self):
 
 
 setup(name='citrine',
-      version='0.30.0',
+      version='0.30.1',
```

Comment:
Or should this be 0.31.0, since it's adapting to breaking changes in GEMD?

Reply:
I'd be conservative and make it 0.31.0 for that reason, since we expect users to reference GEMD classes directly (e.g. for value types).